src/toolbox/methods_category.ts

```diff
@@ -43,6 +40,9 @@ export const getCategory = () => ({
 
 export class MethodsCategory {
   private currentModule: commonStorage.Module | null = null;
+  private robotClassBlocks = getBaseClassBlocks('blocks_base_classes.robot_base.RobotBase');
```

Comment:
Hmmm.   I guess this works now, but if we ever have methods in the base classes that we don't expect to be re-implemented will this cause us a problem?

Reply:
If that happens we'll decide what to do